go/util/receipts/receipts.go

```diff
@@ -6,34 +6,30 @@ package receipts
 
 import (
 	"crypto/rand"
+	"crypto/sha512"
 	"encoding/base64"
 	"fmt"
 	"net/url"
 	"time"
 
 	"github.com/attic-labs/noms/go/d"
-	"github.com/attic-labs/noms/go/hash"
 	"golang.org/x/crypto/nacl/secretbox"
 )
 
 // Data stores parsed receipt data.
 type Data struct {
-	Database string
-	Date     time.Time
+	Database  string
```

Comment:
Are you saying to add a `User` field?

I can imagine doing that, but it's not immediately useful. That's why the query string thing is nice because I can do it later.